library/spark/spark-library/src/test/java/uk/gov/gchq/gaffer/spark/serialisation/kryo/impl/EdgeKryoSerializerTest.java

```diff
@@ -36,7 +36,7 @@ public void setup() {
     }
 
     @Test
-    public void testEdge() {
+    public void testEdgeKryoSerializer() {
```

Comment:
These test methods should begin with the keyword "should" and should describe the expected behaviour of the test, for example:

```java
@Test
public void shouldSerialiseDeserialiseEdge() {
    // Test logic here
}
```